.gitignore

```diff
@@ -7,4 +10,4 @@ _site/
 releases/*.tar.gz
 releases/*.tar.gz.sha256sum
 _output/
-.vagrant/
+.vagrant/
```

Comment:
oh! Looks like you accidentally removed the trailing newline here

![IMG_9149](https://github.com/containerd/containerd/assets/1804568/dff55d9b-87f5-40fd-897d-bc930f9315de)

Reply:
updated. wasn't notice it 😓